### PR TITLE
FetchAll actually returns list of results, not an array per se.

### PR DESCRIPTION
**Pull request by patrickkusebauch**

### CODE REVIEW

File: src/Rules/SyntaxErrorInDibiPreparedStatementMethodRule.php [hunk 1]
```diff
@@ -112,13 +112,7 @@ private function checkErrors(CallLike $callLike, Scope $scope, MethodReflection
             $parameterType = $scope->getType($parameterExpr);
 
             if ($parameterType instanceof StringType) {
-                $resolvedString = $queryReflection->resolveQueryString($parameterExpr, $scope);
```
Comment (staabm) [inline src/Rules/SyntaxErrorInDibiPreparedStatementMethodRule.php:115]: looks like a unrelated change?

Comment (patrickkusebauch) [reply src/Rules/SyntaxErrorInDibiPreparedStatementMethodRule.php:115]: Well, the original method `resolveQueryString` is deprecated, the suggested replacement `resolveQueryStrings` works, but the pass-through of the deprecated method is to a different method that is not doing what is expected and therefore is broken in this use case.

Basically, fixed something that was broken and would not allow any testing to pass in the first place, but was too lazy to do it in a separate stacked PR:

Comment (staabm) [reply src/Rules/SyntaxErrorInDibiPreparedStatementMethodRule.php:115]: I will move it into a separate PR, thank you

**Comment by staabm** [comment]
fyi, this will only work if the list-type is enabled on your projects, which means e.g. bleeding-edge

**Comment by patrickkusebauch** [comment]
Yeah, that's fair.
